site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -177,6 +177,125 @@ spec:
 See the [Webhook Endpoint Specification](#webhook-endpoint-specification) for the specification of the incoming and 
 outgoing JSON packet structure for the webhook endpoint.
 
+## Schedule and Chain Autoscaling
+
+A Schedule based autoscaler can be used to autoscale `GameServers` based on a schedule associated with a policy.
+
+For example, if you have an in-game event happening at 12:00 AM PST on October 31, 2024 and you want to ensure that there are always at least 5 ready game servers at that specific time, you could use a schedule-based autoscaler that applies a buffer policy with a buffer size of 5. 
+
+Schedule-based `FleetAutoscaler` specification below and in the {{< ghlink href="examples/schedulefleetautoscaler.yaml" >}}example folder{{< /ghlink >}}:
+
+```yaml
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: schedule-fleet-autoscaler
+spec:
+  fleetName: fleet-example
+  policy:
+    # Schedule based policy for autoscaling.
+    type: Schedule
+    schedule:
+      between:
+        # The policy becomes eligible for application starting on October 31st, 2024 at 12:00 AM PST. If not set, the policy will immediately be eligible for application.
+        start: "2024-10-31T00:00:00-07:00"
+        # The policy is never ineligible for application. If not set, the policy will always be eligible for application (after the start time).
+        end: ""
+      activePeriod:
+        # Use PST time for the startCron field. Defaults to UTC if not set.
+        timezone: "America/Los_Angeles"
+        # Start applying the policy everyday at 12:00 AM PST. If not set, the policy will always be applied in the .between window.
+        # (Only eligible starting on October 31, 2024 at 12:00 AM PST).
+        startCron: "0 0 * * *"
+        # Apply this policy indefinitely. If not set, the duration will be defaulted to always/indefinite.
+        duration: ""
+      # Policy to be applied during the activePeriod. Required.
+      policy:
+        type: Buffer
+        buffer:
+          bufferSize: 5
+          minReplicas: 10
+          maxReplicas: 20
+  # The autoscaling sync strategy, this will determine how frequent the schedule is evaluated.
+  sync:
+    # type of the sync. for now, only FixedInterval is available
+    type: FixedInterval
+    # parameters of the fixedInterval sync
+    fixedInterval:
+      # the time in seconds between each auto scaling
+      seconds: 30
+```
+
+A Chain based autoscaler can be used to autoscale `GameServers` based on a list of policies, the index of each policy within the list determines the priority in which the policies are evaluated and applied.
```

Comment:
Nit: Add a comment on how the index of each policy within the list determines the priority. I.e. does the first item in the list have the highest priority?